packages/semaphore-identity/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "@zk-kit/semaphore-identity-artifacts",
+    "version": "1.0.0-beta",
```

Comment:
is it exactly the version you want to release?
The workflow I am using is:
1. (manual) `npm run changeset` (to add a changeset md file in the PR)
2. (manual) merge 1st PR
3. (ci) release GH workflow detects the changeset file and opens a PR to publish the package (e.g https://github.com/privacy-scaling-explorations/snark-artifacts/pull/34)
4. (manual) merge this second PR
5. (ci) release GH workflow runs again and publishes the versions not yet published  (e.g. https://github.com/privacy-scaling-explorations/snark-artifacts/actions/runs/9063679321/job/24900130181#step:5:22)

Manually setting a new version in package.json like you're doing here will work too.
It will directly go to step 5 described before. The release workflow will detect that 1.0.0-beta is not yet published and therefore publish it.

A difference when not using the `changeset` cli is that the `CHANGELOG.md` won't be updated...
So for consistency we need to decide to either always use the changeset cli or never use it...
I was precisely thinking about the usefulness of `changeset` for that repo today. `changeset` is very useful to automate describing changes for new versions, updating package.json and publishing...
But given:
- the content of that repo: mostly binary files...does it make sense to describe the changes for a binary file humans can't read... is the CHANGELOG.md really useful?
- we now use the `circuit` package.json field to point to the src files anyway

`changeset` might be overkill here.

Also a limitation of `changeset` is that it does not support prerelease tags (like `-beta`), only `major.minor.patch`.

What do you think?

Reply:
Good points! I've actually tried to run `changeset` and I didn't know what to write there, and I didn't want to release a major, I needed the version I specified in the package.json file. I also think the changelog doesn't make much sense here.

So, we could remove it from this repo if it makes sense for you 👍🏽